src/main/java/uk/gov/cslearning/record/domain/Invite.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.cslearning.record.domain;
+
+import javax.persistence.*;
+
+@Entity
```

Comment:
Can you remove the getters, setters and toString() and use Lombok `@Data` here please?